src/build/Wpf/Wpf.targets

```diff
@@ -0,0 +1,43 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project>
```

Comment:
@onovotny ❓ How do I get this included as one of the .targets files?

Reply:
I would create a `netframwork.targets` in the `\platforms` directory as those will be wild-carded in. Then, in that file, a conditional import based on `TargetFrameworkIdentifier` should work?

Reply:
@onovotny I'm trying to make the default WPF includes behave like the SDK ones. However, it's not clear to me how `<EnableDefaultItems>` defined in the user's project file would impact [**Microsoft.NET.Sdk.DefaultItems.props**](https://github.com/dotnet/sdk/blob/feca75bd43f6374715621024d930164de8bde67c/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.DefaultItems.props#L25-L28), which appears to be imported at the beginning.

Reply:
Not sure I follow? Should be easy enough to use `EnableDefaultItems` as a condition on the ItemGroup. I don't think it matters what order they're in (props/targets) since properties are all evaluated before items, aren't they?

Reply:
> since properties are all evaluated before items, aren't they?

Didn't know this! 😄

Reply:
Yeah, quite handy: https://docs.microsoft.com/en-us/visualstudio/msbuild/comparing-properties-and-items#property-and-item-evaluation-order